client/jetpack-onboarding/summary-next-steps.jsx

```diff
@@ -29,6 +30,18 @@ const NextSteps = ( { siteId, steps } ) => (
 export default localize(
 	connect( ( state, { siteId, siteSlug, siteUrl, translate } ) => {
 		const isConnected = isJetpackSite( state, siteId ); // Will only return true if the site is connected to WP.com
+		const settings = getJetpackOnboardingSettings( state, siteId );
+		const additionalSteps = {};
+		const isBusiness = get( settings, 'siteType' ) === 'business';
+		const wantsWoo = get( settings, 'installWooCommerce' ) === true;
+
+		if ( isBusiness && wantsWoo ) {
+			additionalSteps.STORE = {
+				label: translate( 'Set up your store' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Set up my store!' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-summary-woo-setup-step)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).